ZK_Community_Hub

```diff
@@ -0,0 +1,65 @@
+---
+DIP: TBD
+Title: ZK Community Hub
+Status: Draft
+Themes: Social, Community Involvement
+Tags: Event Production
+Authors: michael@0xparc.org, remy@0xparc.org
+Resources Required: Seating for discussion groups, TV-sized screen, Power Outlets, Signage, Desks and tables, Whiteboards
```

Comment:
```suggestion
Resources Required: Seating for discussion groups, TV-sized screen, Power Outlet, Signage, Desks and tables, Whiteboard
```

Note that the Community Hub production guideline only includes one power outlet and one whiteboard. If you need more, please bring it! We recommend bringing extension cords / multi-plugs!